.gitignore

```diff
@@ -16,6 +16,10 @@ typechain-types/
 /coverage
 /coverage.json
 
+# vitest coverage files
+/webapp/coverage/
```

Comment:
In the best case we should upload them somewhere. Maybe worth creating a follow up issue to use codecov (or some similar tool)

Reply:
If we want to collect a lot of experience: https://github.com/codecov/self-hosted 😅

Reply:
More realistically speaking: https://coveralls.io/pricing